src/Symfony/Component/Yaml/Tests/YamlTest.php

```diff
@@ -24,6 +25,37 @@ public function testParseAndDump()
         $this->assertEquals($data, $parsed);
     }
 
+    public function testParseWithMultilineQuotes()
+    {
+        $yaml = <<<YAML
+foo:
+  bar: 'baz
+
+'
```

Comment:
This is not valid YAML AFAIU. The quote must be indented.

Reply:
Then we should close this PR and the issue with "Won't fix", because this is exactly the case, what is requested to be fixed!

Reply:
If I am not mistaken, this is indeed valid YAML.

Reply:
Hi @xabbuh and @fabpot,

I have taken some time to read up on this YAML topic. In the internet are two kinds of YAML parser for this topic. Either the content must be indented even if it is in quotes or everything in quotes can be indented free, because the part in quotes is simply considered as a text block.

What I want to say is that the PR should not be merged as it is and we should decide here how to proceed.

Solution A: everything should be how it is and we only support indented quotes, seems to be how it's described in the spec [https://yaml.org/spec/1.2.2/#8111-block-indentation-indicator](https://yaml.org/spec/1.2.2/#8111-block-indentation-indicator).
```
foo:
  bar: 'baz

'
```

Solution B: be a bit more generous and allow what requested in the issue and how a lot of online parsers/validators see the quoted part as a string and allow stuff like that:
```
foo:
  bar: 'baz1
baz2
                baz3

'
```

Based on that I'd prefer solution A and keep it how it is as @fabpot mentioned. What do you think?